src/components/AddEvent.js

```diff
@@ -71,7 +71,8 @@ const AddEvent = (props) => {
         />
       </div>
       <RaisedButton
-        className="addEventButton primaryButton"
+        primary
+        className="mui-button primary"
```

Comment:
What is this `primary` class used for?

Reply:
@bill-feng Thanks for noticing! I removed it!